src/System.Private.CoreLib/shared/System/Runtime/InteropServices/VariantWrapper.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
ditto

Reply:
This one I'm not sure there's a legitimate reason for `obj` to be null.  @AaronRobinsonMSFT, do you know?

Reply:
`VT_EMPTY` is a valid variant type. So I would say yes it is legit.